olexandr.vakulinskiy/HW2/Task1/Tamagotchi.rb

```diff
@@ -0,0 +1,109 @@
+class Tamagotchi
```

Comment:
file name must be lowercase, [separated by snake case if necessary](https://github.com/rubocop-hq/ruby-style-guide#snake-case-files)